### PR TITLE
add guides to setup vscode

### PR DESCRIPTION
**Pull request by aq1018**

### CODE REVIEW

**Review by acostanzo — APPROVED** [review]
Might be worth a look from another @BuoySoftware/engineers that uses VSCode but I like the idea of having these guides. I'll try and find some time add something for Nvim

File: editor-setup/vscode.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# Recommended VSCode Configurations
+
+## BuoyRails
```
Comment (dgalarza) [inline editor-setup/vscode.md:3]: These repo specific mentions feel a little out of place IMO. I wonder if they fit better in a repo's wiki.

Comment (acostanzo) [reply editor-setup/vscode.md:3]: Oh good catch, I overlooked that. Wonder if the guide could be tweaked to stay with suggestions on helpful plugins removing the reference to the repos

Comment (aq1018) [reply editor-setup/vscode.md]: I added a commit below and made the titles more general. Please take a look.

**Pull request merged**